### PR TITLE
[STAL-3162] Add Kotlin rulesets

### PR DESCRIPTION
**Pull request by rjcoulter22**
### What does this PR do? What is the motivation?
This adds the new kotlin rule sets to the main documentation for the static analysis rules documentation
### Merge instructions

```
/merge
```

### CODE REVIEW

File: content/en/code_analysis/static_analysis_rules/_index.md [hunk 1]
```diff
@@ -91,6 +91,14 @@ rulesets:
     title: "React specific linting rules"
     description: |
       This plugin exports a `recommended` configuration that enforces React good practices.
+  kotlin-best-practices:
+    title: "Rules to enforce Kotlin best practices."
+    description: |
+
```
Comment (dastrong) [inline content/en/code_analysis/static_analysis_rules/_index.md:97]: Ruleset description needed.

File: content/en/code_analysis/static_analysis_rules/_index.md [hunk 2]
```diff
@@ -91,6 +91,14 @@ rulesets:
...
+  kotlin-code-style:
+    title: "Rules to check code style for Kotlin"
+    description: |
+
```
Comment (dastrong) [inline content/en/code_analysis/static_analysis_rules/_index.md:101]: Ruleset description needed.

**Review by dastrong — CHANGES_REQUESTED** [review]
Missing description for the rulesets

**Review by dastrong — APPROVED** [review]

**Comment by rjcoulter22** [comment]
This should not be approved until the following PR is approved/merged

https://github.com/DataDog/datadog-static-analyzer-rule-docs/pull/93

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**